compiler/rustc_resolve/src/rustdoc.rs

```diff
@@ -136,8 +136,11 @@ pub fn unindent_doc_fragments(docs: &mut [DocFragment]) {
                     // Compare against either space or tab, ignoring whether they are
                     // mixed or not.
                     let whitespace = line.chars().take_while(|c| *c == ' ' || *c == '\t').count();
-                    cmp::min(min_indent, whitespace)
-                        + if fragment.kind == DocFragmentKind::SugaredDoc { 0 } else { add }
+                    cmp::min(
+                        min_indent,
+                        whitespace
+                            + (if fragment.kind == DocFragmentKind::SugaredDoc { 0 } else { add }),
+                    )
                 }
             })
```

Comment:
This looks good to me. Could you turn the `fold()` into something like `.filter(is_all_whitespace).map(count_whitespace+add).min()` now that that's possible?

Reply:
Fixed.